spring-kafka/src/main/java/org/springframework/kafka/support/serializer/JsonDeserializer.java

```diff
@@ -196,6 +249,10 @@ else if (configs.get(VALUE_DEFAULT_TYPE) instanceof String) {
 						StringUtils.commaDelimitedListToStringArray((String) configs.get(TRUSTED_PACKAGES)));
 			}
 		}
+		if (configs.containsKey(TYPE_MAPPINGS)) {
```

Comment:
The `JsonSerializer` has more condition in similar place.
Why is it missed here?

Thanks